### DISCUSSION TITLE
Can the model editor map be displayed on the web interface?

### DISCUSSION THREAD

**Discussion by CSXYA** [openTCS/opentcs#87] [Q&A] [answered]
Can the model editor map be displayed on the web interface?

**Comment by swltr** [comment]
Which web interface do you mean?

**Reply by CSXYA** [reply]
browser

**Reply by swltr** [reply, marked as answer]
The openTCS distribution does not contain any web-based UI.

**Reply by CSXYA** [reply]
OK, thank you
